src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="body">
+    <main class="calendar calendar--mounth--mounth29">
+      <div class="day day--day--sunday"></div>
+      <div class="day"></div>
+      <div class="day"></div>
+      <div class="day"></div>
+      <div class="day"></div>
```

Comment:
if you give these divs classes, let's name them according to BEM -> they're elements of calendar block